pkg/backend/openshiftcluster.go

```diff
@@ -81,11 +82,22 @@ func (ocb *openShiftClusterBackend) handle(ctx context.Context, log *logrus.Entr
 		return ocb.endLease(ctx, log, stop, doc, api.ProvisioningStateFailed, err)
 	}
 
+	// Get subscription document so that the list of registered features
+	// it contains can be passed along to Create and AdminUpdate
+	resource, err := azure.ParseResourceID(doc.OpenShiftCluster.ID)
+	if err != nil {
+		return err
+	}
+	subscriptionDoc, err := ocb.db.Subscriptions.Get(ctx, resource.SubscriptionID)
+	if err != nil {
+		return err
+	}
+
```

Comment:
@olga-mir is doing something very similar in https://github.com/Azure/ARO-RP/pull/871 - please follow her route and then start storing subscriptionDoc in `type Manager struct`.